### PR TITLE
More env vars

### PR DESCRIPTION
**Pull request by nickbroon**
This adds 4 useful environment variables related to the ability to write/control Stash forked repo.
Includes a small minor improvement to warning message.

### CODE REVIEW

File: src/main/java/com/ngs/stash/externalhooks/hook/ExternalPreReceiveHook.java [hunk 1]
```diff
@@ -197,7 +197,7 @@ public void validate(
             if (!permissions.hasGlobalPermission(
                     authCtx.getCurrentUser(), Permission.SYS_ADMIN)) {
                 errors.addFieldError("exe",
-                    "You should be an Administrator to edit this field " +
+                    "You should be a Stash System Administrator to edit this field " +
```
Comment (seletskiy) [inline src/main/java/com/ngs/stash/externalhooks/hook/ExternalPreReceiveHook.java]: Stash is now Bitbucket, so message should be changed accordingly.

Comment (nickbroon) [inline src/main/java/com/ngs/stash/externalhooks/hook/ExternalPreReceiveHook.java]: I've corrected this.

**Comment by seletskiy** [comment]
@nickbroon: Thanks! Great contribution!

At last, can you change version number in the `pom.xml` file?

I'll upload new build onto marketplace little bit after.

File: pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.ngs.stash.externalhooks</groupId>
     <artifactId>external-hooks</artifactId>
-    <version>3.0-1</version>
+    <version>3.0-2</version>
```
Comment (seletskiy) [inline pom.xml]: Please, set `<version>` to 3.2-1. I've had some un-pushed changes till now :(

**Comment by nickbroon** [comment]
Version bumped, and merge conflict resolved.

**Comment by seletskiy** [comment]
@nickbroon: Great! Thanks for the contribution!

**Comment by nickbroon** [comment]
No probs, thanks.
Will you release 3.2-1 to the Marketplace soon?

**Comment by seletskiy** [comment]
@nickbroon: I want to get feedback on #31 so I do not need to update package twice. If there will be no reply in a week I'll update it anyway.

**Comment by nickbroon** [comment]
That's great, thanks. :-)
The sooner it appears in the Market Place the sooner I get our sys-ops team to deploy.

**Comment by seletskiy** [comment]
@nickbroon: 3.3 which contains your changes is released on marketplace.

**Pull request merged**